conformance/setup.go

```diff
@@ -96,46 +100,60 @@ var (
-	layerBlobDigest            string
-	layerBlobContentLength     string
-	emptyLayerManifestContent  []byte
-	nonexistentManifest        string
-	reportJUnitFilename        string
-	reportHTMLFilename         string
-	httpWriter                 *httpDebugWriter
-	testsToRun                 int
-	suiteDescription           string
-	runPullSetup               bool
-	runPushSetup               bool
-	runContentDiscoverySetup   bool
-	runContentManagementSetup  bool
-	skipEmptyLayerTest         bool
-	deleteManifestBeforeBlobs  bool
-	runAutomaticCrossmountTest bool
-	automaticCrossmountEnabled bool
-	configs                    []TestBlob
-	manifests                  []TestBlob
-	seed                       int64
-	Version                    = "unknown"
+	testBlobA                         []byte
+	testBlobALength                   string
+	testBlobADigest                   string
+	testRefBlobA                      []byte
+	testRefBlobALength                string
+	testRefBlobADigest                string
+	testBlobB                         []byte
+	testBlobBDigest                   string
+	testBlobBChunk1                   []byte
+	testBlobBChunk1Length             string
+	testBlobBChunk2                   []byte
+	testBlobBChunk2Length             string
+	testBlobBChunk1Range              string
+	testBlobBChunk2Range              string
+	client                            *reggie.Client
+	crossmountNamespace               string
+	dummyDigest                       string
+	errorCodes                        []string
+	invalidManifestContent            []byte
+	layerBlobData                     []byte
+	layerBlobDigest                   string
+	layerBlobContentLength            string
+	emptyLayerManifestContent         []byte
+	refsManifestConfigArtifactContent []byte
+	refsManifestConfigArtifactDigest  string
+	refsManifestLayerArtifactContent  []byte
+	refsManifestLayerArtifactDigest   string
+	nonexistentManifest               string
+	reportJUnitFilename               string
+	reportHTMLFilename                string
+	httpWriter                        *httpDebugWriter
+	testsToRun                        int
+	suiteDescription                  string
+	runPullSetup                      bool
+	runPushSetup                      bool
+	runContentDiscoverySetup          bool
+	runContentManagementSetup         bool
+	runReferencesSetup                bool
+	skipEmptyLayerTest                bool
+	deleteManifestBeforeBlobs         bool
+	runAutomaticCrossmountTest        bool
+	automaticCrossmountEnabled        bool
+	configs                           []TestBlob
+	manifests                         []TestBlob
+	seed                              int64
+	Version                           = "unknown"
+	scratchDescriptor                 = Descriptor{
+		MediaType: "application/vnd.oci.scratch.v1+json",
+		Size:      2,
+		Digest:    godigest.FromString("sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"),
+	}
```

Comment:
Pending the result of https://github.com/opencontainers/image-spec/pull/1068

Reply:
Used same empty JSON descriptor